media-fonts/ocr/additional-files/fontconfig-relaod-cache.sh

```diff
@@ -0,0 +1,3 @@
+#!sh
```

Comment:
```suggestion
#!/bin/sh
```

Reply:
Got a local change for that already, will push later :)

Reply:
Fixed